slicing_copy.lua

```diff
@@ -104,19 +108,19 @@ local function cut(shift, endpos)
         :arg("-v", "warning")
         :arg(o.overwrite and "-y" or "-n")
         :arg("-stats")
-    if ua and ua ~= '' and ua ~= 'libmpv' then
-        cmds:arg('-user_agent', ua)
+    if is_remote() and ua and ua ~= '' and ua ~= 'libmpv' then
+        cmds:arg('-user_agent', "\""..ua.."\"")
```

Comment:
```suggestion
        cmds:arg('-user_agent', ua)
```

This is incorrect because the quotes will be passed as part of the UA.

Below is a simple test.
```
09:47:11 [INFO] 127.0.0.1 "GET /[HYSUB]Jashin-chan%20Dropkick%20X[01v2][GB_MP4][1920X1080].mp4 HTTP/1.1" 200 5832704 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:104.0) Gecko/20100101 Fir
efox/104.0" 0.023854
09:47:11 [INFO] 127.0.0.1 "GET /[HYSUB]Jashin-chan%20Dropkick%20X[01v2][GB_MP4][1920X1080].mp4 HTTP/1.1" 206 75 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:104.0) Gecko/20100101 Firefox/
104.0" 0.001083
09:47:18 [INFO] 127.0.0.1 "GET /[HYSUB]Jashin-chan%20Dropkick%20X[01v2][GB_MP4][1920X1080].mp4 HTTP/1.1" 206 75 "-" ""Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:104.0) Gecko/20100101 Firefox
/104.0"" 0.000416
09:47:18 [INFO] 127.0.0.1 "GET /[HYSUB]Jashin-chan%20Dropkick%20X[01v2][GB_MP4][1920X1080].mp4 HTTP/1.1" 200 1966080 "-" ""Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:104.0) Gecko/20100101 Fi
refox/104.0"" 0.011121
```

Reply:
I have user-agent set in config with quotes, below is fragment of my `mpv.conf`
```
user-agent="Mozilla/5.0 (Windows NT 10.0; rv:99.0) Gecko/20100101 Firefox/99.0"
```

With `cmds:arg('-user_agent', "\""..ua.."\"")` result ffmpeg command look like this
```
... -stats -user_agent "Mozilla/5.0 (Windows NT 10.0; rv:99.0) Gecko/20100101 Firefox/99.0" -ss ....
```
but with `cmds:arg('-user_agent', ua)` command look like this
```
... -stats -user_agent Mozilla/5.0 (Windows NT 10.0; rv:99.0) Gecko/20100101 Firefox/99.0 -ss ...
```

Reply:
The below one is correct.

With above one the UA is `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:104.0) Gecko/20100101 Firefox/104.0"` (the server will see extra quotes), while it should be `Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:104.0) Gecko/20100101 Firefox/104.0`.

Reply:
Well, that's weird.